src/main/resources/templates/fragments/inputs/submitButtonSecondary.html

```diff
@@ -0,0 +1,9 @@
+<th:block
```

Comment:
This one I'm not sure if we should add or not. What do you think?  Is it worth having a separate one just for the secondary color or should we have a way to pass in style, which seems odd as well?

Reply:
Doesn't hurt, makes it easier to use since the first hard codes the button--primary class.